### ISSUE TITLE
Invalid `toString()` method generated for modifiable class

### ISSUE THREAD

**Issue by mseitner** [immutables/immutables#536]
I have a relatively simple class PostData annotated with `@Value.Modifiable`.

```java
@Value.Immutable
@Value.Modifiable
@Value.Style(
        get = { "get*", "is*"},
        validationMethod = Value.Style.ValidationMethod.NONE
)
@JsonDeserialize(as = ImmutablePostData.class)
public interface PostData {

    @JsonProperty(value = "login_allowed")
    boolean isLoginAllowed();

    // some other stuff, nothing fancy

}
```

The ImmutablePostData class generates fine, but ModifiablePostData has an error. The relevant section of that generated file is:

```java
@Override
public String toString() {
return "ModifiablePostData{"
	+ "loginAllowed="  + (loginAllowedIsSet() ? isLoginAllowed() : "?")
	// other fields
	+ "}";
}
```

The method `loginAllowedIsSet()` was not generated, which is probably correct, because due to `validationMethod = Value.Style.ValidationMethod.NONE` that check should not be necessary.

- Immutables 2.4.2
- Oracle JDK 1.8.0_121

**Comment by elucash** [comment]
fixed in the master/latest snapshot. will be released in 2.4.3. Thank you!

**Comment by elucash** [comment]
Fix released in 2.4.3, thank you!
